x-pack/plugin/security/qa/security-trial/src/javaRestTest/java/org/elasticsearch/xpack/security/crossclusteraccess/CrossClusterAccessHeadersForCcsRestIT.java

```diff
@@ -929,7 +925,8 @@ private void expectActionsAndHeadersForCluster(
                 // this action is run by the cross cluster access user, so we expect a cross cluster access header with an internal user
                 // authentication and pre-defined role descriptors intersection
                 case RemoteClusterNodesAction.NAME -> {
-                    assertContainsCrossClusterAccessHeaders(actual.headers());
+                    // internal users don't include an audit request ID; this is a current limitation of the audit setup
```

Comment:
I don't really want to mess with this in the context of RCS 2.0. We could generate a fresh request ID for internal user requests but that introduces a new subtle inconsistency. Also don't feel to strongly but slight preference to keep things minimal for now.

Reply:
This sounds fine to me. If the local request does not have an auditID, it means it will not have an audit entry. So there is nothing in the local cluster to correlate with.